src/components/KCheckbox/KCheckbox.vue

```diff
@@ -17,70 +17,75 @@
       <slot>{{ label }}</slot>
     </span>
     <div
-      v-if="hasLabel && (description || $slots.description)"
+      v-if="showDescription"
       class="k-checkbox-description"
     >
       <slot name="description">{{ description }}</slot>
     </div>
   </label>
 </template>
 
-<script lang="ts">
-import { defineComponent, computed } from 'vue'
+<script lang="ts" setup>
+import { computed, useAttrs, useSlots } from 'vue'
 
-export default defineComponent({
-  name: 'KCheckbox',
-  inheritAttrs: false,
-  props: {
-    /**
-     * Sets whether or not checkbox is checked
-     */
-    modelValue: {
-      type: Boolean,
-      default: false,
-      required: true,
-    },
-    /**
-     * Overrides default label text
-     */
-    label: {
-      type: String,
-      default: '',
-    },
-    /**
-     * Overrides default description text
-     */
-    description: {
-      type: String,
-      default: '',
-    },
+const props = defineProps({
+/**
+* Sets whether or not checkbox is checked
+*/
```

Comment:
nit: indentation formatting on all the prop comments

Reply:
fixed